### PR TITLE
[WIP] TWW Support

### PR DESCRIPTION
**Pull request by ls-**
In TWW Blizz added this new thingy, the temporary max health loss bar. 

image
In this pic the max health was reduced by 50%, but the player is still at 100% health, it's just the total available amount of hp that's reduced temporarily, my assumption so far is that it's due to some raid/dungeon mechanics/auras hence the "temporary" bit in its name. This bar is kinda similar to "Reserved Life" from Path of Exile.

### CODE REVIEW

**Comment by ls-** [comment]
K, after a heated discussion we decided to move it to the health element.

**Comment by ls-** [comment]
K, this PR is now the TWW PR, all the TWW-related work will be happening here. I won't be renaming the git branch, it's whatever 😒

File: elements/powerprediction.lua [hunk 1]
```diff
@@ -43,12 +43,25 @@ A default texture will be applied if the widget is a StatusBar and doesn't have
 local _, ns = ...
 local oUF = ns.oUF
 
--- sourced from FrameXML/AlternatePowerBar.lua
+-- sourced from Blizzard_UnitFrame/AlternatePowerBar.lua
 local ALT_POWER_BAR_PAIR_DISPLAY_INFO = _G.ALT_POWER_BAR_PAIR_DISPLAY_INFO
+
```
Comment (ls-) [inline elements/powerprediction.lua:48]: Why did add an empty line here tho?

File: elements/classpower.lua [hunk 2]
```diff
@@ -49,10 +50,11 @@ local oUF = ns.oUF
 
 local _, PlayerClass = UnitClass('player')
 
--- sourced from FrameXML/Constants.lua
+-- sourced from Blizzard_FrameXMLBase/Constants.lua
 local SPEC_MAGE_ARCANE = _G.SPEC_MAGE_ARCANE or 1
 local SPEC_MONK_WINDWALKER = _G.SPEC_MONK_WINDWALKER or 3
 local SPEC_WARLOCK_DESTRUCTION = _G.SPEC_WARLOCK_DESTRUCTION or 3
+
```
Comment (ls-) [inline elements/classpower.lua:57]: And here?

File: elements/alternativepower.lua [hunk 3]
```diff
@@ -56,8 +56,9 @@ local Private = oUF.Private
 
 local unitSelectionType = Private.unitSelectionType
 
--- sourced from FrameXML/UnitPowerBarAlt.lua
+-- sourced from Blizzard_UnitFrame/UnitPowerBarAlt.lua
 local ALTERNATE_POWER_INDEX = Enum.PowerType.Alternate or 10
+
```
Comment (ls-) [inline elements/alternativepower.lua:61]: And here?

File: elements/additionalpower.lua [hunk 4]
```diff
@@ -59,8 +59,9 @@ local oUF = ns.oUF
 
 local _, playerClass = UnitClass('player')
 
--- sourced from FrameXML/AlternatePowerBar.lua
+-- sourced from Blizzard_UnitFrame/AlternatePowerBar.lua
 local ALT_POWER_BAR_PAIR_DISPLAY_INFO = _G.ALT_POWER_BAR_PAIR_DISPLAY_INFO
+
```
Comment (ls-) [inline elements/additionalpower.lua:64]: And here?

**Review by ls- — COMMENTED** [review]
Weird empty lines 🤔

**Review by p3lim — APPROVED** [review]

**Pull request merged**